commands/Tools/todo.js

```diff
@@ -0,0 +1,54 @@
+/*
+To your main server file, add:
+Client.defaultUserSchema
+  .add("TODOs", "any", {array: true, default: [], configurable: true});
```

Comment:
```suggestion
  .add('TODOs', 'any', { array: true });
```

When `array` is set to true, `default` becomes an empty array by default, and when type is `any`, `configurable` is always set to false.